### PR TITLE
HTML-15 - oddNumbers

### PR DESCRIPTION
**Pull request by zeljkosoko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Odd numbers</title>
+</head>
+<body>
+	<script src="oddNumbers.js"></script>
+</body>
+</html>
```
Comment (ban17) [inline index.html]: Always add empty line at the end of files.

File: oddNumbers.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+oddNumbers = () => {
```
Comment (ban17) [inline oddNumbers.js]: var,let or const missing

File: index.html (hunk 1 above)
Comment (zeljkosoko) [reply index.html]: Understand.

**Review by hanii1993 — APPROVED** [review]

File: oddNumbers.js (hunk 2 above)
Comment (zeljkosoko) [reply oddNumbers.js]: script is changed in new commit.

**Review by ban17 — APPROVED** [review]

**Pull request merged**